lib/client-assets.php

```diff
@@ -236,15 +236,14 @@ function gutenberg_register_packages_scripts( $scripts ) {
 	// When in production, use the plugin's version as the default asset version;
 	// else (for development or test) default to use the current time.
 	$default_version = defined( 'GUTENBERG_VERSION' ) && ! ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) ? GUTENBERG_VERSION : time();
-	$suffix          = '.min.js';
 
-	foreach ( glob( gutenberg_dir_path() . "build/*/index$suffix" ) as $path ) {
+	foreach ( glob( gutenberg_dir_path() . 'build/*/index.min.js' ) as $path ) {
 		// Prefix `wp-` to package directory to get script handle.
 		// For example, `…/build/a11y/index.min.js` becomes `wp-a11y`.
 		$handle = 'wp-' . basename( dirname( $path ) );
 
 		// Replace suffix and extension with `.asset.php` to find the generated dependencies file.
```

Comment:
comment nit: now we no longer "replace suffix and extension", but just "replace extension"

Reply:
```suggestion
		// Replace extension with `.asset.php` to find the generated dependencies file.
```